src/diffusers/loaders.py

```diff
@@ -1099,6 +1104,8 @@ def lora_scale(self) -> float:
 
     def _remove_text_encoder_monkey_patch(self):
         self._remove_text_encoder_monkey_patch_classmethod(self.text_encoder)
+        if hasattr(self, "text_encoder_2"):
+            self._remove_text_encoder_monkey_patch_classmethod(self.text_encoder_2)
```

Comment:
let's move the sdxl pipeline specific code to the sdxl pipeline

i.e.

```py
class SDXLPipeline:
    def _remove_text_encoder_monkey_patch(self):
         self._remove_text_encoder_monkey_patch_classmethod(self.text_encoder)
         self._remove_text_encoder_monkey_patch_classmethod(self.text_encoder_2)
```

and then we don't have to touch the _remove_text_encoder_monkey_patch method in the general mixin